Update to therapeutics (#49)
greenelab/covid19-review: content/20.treatments.md

Second, the development of antiviral drugs can drive a reduced recovery time for patients by inhibiting the development of the virus once an individual is infected.
Finally, vaccines present a strategy for bolstering the immune response of the populus broadly to the virus, resulting in a lower rate of infection.
All three of these strategies have been valuable elements of responses to other viruses, including coronaviruses, and are being investigated by researchers at present.

> Additionally, there has been suggestions amongst the scientific community that nutraceutical or dietary supplement interventions may exhibit efficacy to prime an individual’s immune system to prevent or lessen the impact of RNA virus infections [@doi:10.1016/j.pcad.2020.02.007; @doi:10.1016/S0140-6736(20)30305-6]. 

Reviewer: ```suggestion
Additionally, there have been suggestions within the scientific community that nutraceutical or dietary supplement interventions may prime an individual’s immune system to prevent or lessen the impact of RNA virus infections [@doi:10.1016/j.pcad.2020.02.007; @doi:10.1016/S0140-6736(20)30305-6]. 
```

> Additionally, there has been suggestions amongst the scientific community that nutraceutical or dietary supplement interventions may exhibit efficacy to prime an individual’s immune system to prevent or lessen the impact of RNA virus infections [@https://doi.org/10.1016/j.pcad.2020.02.007; https://doi.org/10.1016/S0140-6736(20)30305-6]. 

Reviewer 2: ```suggestion
Additionally, there has been suggestions amongst the scientific community that nutraceutical or dietary supplement interventions may exhibit efficacy to prime an individual’s immune system to prevent or lessen the impact of RNA virus infections [@doi:10.1016/j.pcad.2020.02.007; @doi:10.1016/S0140-6736(20)30305-6]. 
```
